components/server/src/ome/services/RenderingBean.java

```diff
@@ -552,22 +553,13 @@ public RGBBuffer render(PlaneDef pd) {
             sizeY = sizeY/stride;
             BufferedImage image = ImageUtil.createBufferedImage(buf, sizeX,
                     sizeY);
-            byteStream = new ByteArrayOutputStream();
             compressionSrv.compressToStream(image, byteStream);
             return byteStream.toByteArray();
         } catch (IOException e) {
             log.error("Could not compress rendered image.", e);
             throw new ResourceError(e.getMessage());
         } finally {
             rwl.writeLock().unlock();
-            try {
-                if (byteStream != null) {
-                    byteStream.close();
-                }
-            } catch (IOException e) {
-                log.error("Could not close byte stream.", e);
-                throw new ResourceError(e.getMessage());
```

Comment:
I did not want to change the behaviour of the existing method i.e. ``projectPixels``